tests/subsys/fs/lib_link/app.overlay

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
```

Comment:
Nit: Copyright

Reply:
Heh. What is wrong with it?

Reply:
So this actually wasn't an oversight, since I simply copied this file from a different part of the repo, I left the copyright as it was.
Should I be updating that if the only action I took was the copy?

Reply:
> So this actually wasn't an oversight, since I simply copied this file from a different part of the repo, I left the copyright as it was.

ok.